.changeset/thick-spiders-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/autocomplete": patch
+---
+
+Fixing the type of AutocompleteItemWithItems and adding labelProps.
```

Comment:
Please correct this description as well.

https://github.com/yamada-ui/yamada-ui/pull/2760/files#r1762265693

Reply:
fixed